### PR TITLE
fetch and parse OR getvaccinated site list

### LINKED ISSUE #714: fetch or/getvaccinated_gov
[learn our pipeline: fetch](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Runner-pipeline-stages#fetch)

Fetch data from: https://getvaccinated.oregon.gov/get_published_events

Put your script in a file named: `or/getvaccinated_gov/fetch.sh`

Store it (without processing) in a new file created in the directory passed as the first argument (`sys.argv[1]`).

Check the wiki to learn more about the purpose of the fetch stage and how to get set up for development!

## Notes:

this is the data source behind https://getvaccinated.oregon.gov/#/locator?lang=en and doesnt seem to be part of the scraper yet. 

It probably doesnt contain much new information TBH

### Tips

1. While working on your code, run it at any point:
    ```sh
    poetry run vaccine-feed-ingest fetch <state>/<site>
    ```

### PR DESCRIPTION
**Pull request by MoralCode**
# Fetch and parse or/getvaccinated_gov

| Key Details |
|-|
| Resolves #714  |
State: or |
Site: getvaccinated_gov |

## Data sample

```json
{"event_eligibility_questions": [], "doses": [{"qty2": "", "qty1": 500, "brand": "Pfizer"}, {"brand": "", "qty1": "", "qty2": ""}, {"brand": "", "qty1": "", "qty2": ""}], "longitude": -122.7739871, "status_updated_by": {"email": "carrie_beck@co.washington.or.us", "loc_primary_lname": "Beck", "loc_primary_fname": "Carrie", "providers": [{"provider_id": "ORAL1616A1", "has_access": true}]}, "event_city": "Tualatin", "event_counties": ["Washington"], "additional_information": "Drive-thru clinic but can also accept walk-ups.  Scheduled appointments encouraged but will also accept drop-ins.  Neighborhood Health Center is the Vaccinator. Call 503-848-5847 for scheduling and additional questions.", "minor_consent": "true", "event_ages": ["80+", "75-80", "70-75", "65-70", "45-65", "18-45", "16-18", "12-16"], "is_closed_event": false, "event_status": "Approved - Published", "is_recurrent_event": false, "approved_ts": "2021-06-02", "event_end_time": "5:00 PM", "event_location": "Tualatin High School, North Parking Lot", "provider_name": "WASHINGTON CO - HILLSBORO CLINIC", "event_start_time": "9:00 AM", "event_date": "06-05-2021", "event_street_address": "22300 Boones Ferry Rd", "event_name": "Neighborhood Health Center - Tigard/Tualatin School District - Pfizer Drive Thru Clinic", "ada_recommendations": "ADA Accessible", "poc_email": "Drew.Hawk@dhsoha.state.or.us", "eligibility_status": "", "provider": {"org_name": "WASHINGTON CO - HILLSBORO CLINIC", "loc_admin_street1": "155 N 1ST AVE STOP 68", "loc_admin_city": "Hillsboro", "org_phone": "971-330-1776", "loc_name": "WASHINGTON CO - HILLSBORO CLINIC", "loc_admin_county": "Washington", "loc_covid_id": "ORAL1616A1", "loc_admin_phone": "971-330-1776", "is_disabled": false, "created_date": "03-26-2021", "loc_admin_state": "WA", "loc_admin_zip": "97124"}, "event_id": "c0d1dabf-f978-4454-892d-0512a094f968", "latitude": 45.3590865, "event_link_or_phone": "503-848-5847 - Neighborhood Health Center Scheduling", "published_ts": "2021-06-02", "notify_people": "Yes", "additional_contacts": "Neighborhood Health Center - 503-848-5847", "event_created_date": "06-02-2021", "language_interpretation_service": "Available as Needed", "event_zip_code": 97062, "recurrent_day": ""}
{"eligibility_status": "", "event_id": "b5dd78a9-f730-4113-8667-229e4293658a", "event_eligibility_questions": [], "event_end_time": "7:00 PM", "event_name": "Columbia Sportswear Drive-Thru Pfizer Clinic 6/6, 6/7", "poc_email": "Drew.Hawk@dhsoha.state.or.us", "ada_recommendations": "ADA Accessible", "minor_consent": "true", "event_counties": ["Washington"], "event_location": "Columbia Sportswear", "recurrent_day": "", "event_street_address": "14375 NW Science Park Dr", "notify_people": "Yes", "latitude": 45.5302655, "provider_name": "WASHINGTON CO - HILLSBORO CLINIC", "additional_information": "Registration for this event is required.  Schedule an appointment using the website link.  ", "status_updated_by": {"email": "carrie_beck@co.washington.or.us", "loc_primary_lname": "Beck", "loc_primary_fname": "Carrie", "providers": [{"has_access": true, "provider_id": "ORAL1616A1"}]}, "approved_ts": "2021-06-02", "is_closed_event": false, "published_ts": "2021-06-02", "event_start_time": "1:00 PM", "event_ages": ["80+", "75-80", "70-75", "65-70", "45-65", "18-45", "16-18", "12-16"], "event_city": "Portland", "longitude": -122.8246245, "provider": {"created_date": "03-26-2021", "loc_covid_id": "ORAL1616A1", "loc_admin_zip": "97124", "loc_admin_street1": "155 N 1ST AVE STOP 68", "org_phone": "971-330-1776", "loc_name": "WASHINGTON CO - HILLSBORO CLINIC", "loc_admin_state": "WA", "loc_admin_phone": "971-330-1776", "is_disabled": false, "loc_admin_city": "Hillsboro", "org_name": "WASHINGTON CO - HILLSBORO CLINIC", "loc_admin_county": "Washington"}, "event_status": "Approved - Published", "event_date": "06-06-2021", "is_recurrent_event": false, "language_interpretation_service": "Services available as needed", "doses": [{"qty1": 500, "qty2": "", "brand": "Pfizer"}, {"qty2": "", "brand": "", "qty1": ""}, {"qty1": "", "brand": "", "qty2": ""}], "event_created_date": "06-02-2021", "event_zip_code": 97229, "additional_contacts": "Carrie Beck, carrie_beck@co.washington.or.us, 971-330-1776", "event_link_or_phone": "http://kordinator.mhealthcoach.net/vcl/ColumbiaSportswearHQ"}
{"event_end_time": "11:30 AM", "event_id": "13ac060a-3297-4923-9dda-afc1b0d8dbbf", "longitude": -122.6762003, "event_start_time": "9:00 AM", "event_street_address": "601 Southwest 4th Avenue", "approved_ts": "2021-06-03", "doses": [{"brand": "Johnson & Johnson", "qty2": "", "qty1": 100}, {"qty1": "", "brand": "", "qty2": ""}, {"qty1": "", "brand": "", "qty2": ""}], "recurrent_day": "", "event_ages": ["80+", "75-80", "70-75", "65-70", "45-65", "18-45"], "provider_name": "ONE MEDICAL DOWNTOWN", "event_date": "06-07-2021", "additional_contacts": "Lori Bolsinger - loribolsinger@onemedical.com\nMichael Chen - mchen@onemedical.com", "event_status": "Approved - Published", "is_closed_event": false, "additional_information": "By appointment only, please use the website link to make an appointment", "is_recurrent_event": false, "event_created_date": "06-03-2021", "latitude": 45.5191903, "notify_people": "Yes", "language_interpretation_service": "Language Interpretation Services Available", "status_updated_by": {"loc_primary_fname": "Michael", "email": "mchen@onemedical.com", "loc_primary_lname": "Chen", "providers": [{"provider_id": "ORAL4924A1", "has_access": true}, {"has_access": true, "provider_id": "ORAL4844A2"}, {"provider_id": "ORAL4818A1", "has_access": true}]}, "eligibility_status": "Eligible", "event_city": "Portland", "poc_email": "Drew.Hawk@dhsoha.state.or.us", "event_counties": ["Clackamas", "Multnomah", "Washington"], "event_link_or_phone": "https://www.onemedical.com/omvax/", "event_location": "One Medical Downtown", "published_ts": "2021-06-04", "event_eligibility_questions": [], "event_name": "J&J COVID-19 Vaccine", "ada_recommendations": "ADA Accessible", "minor_consent": "null", "provider": {"org_name": "ONE MEDICAL DOWNTOWN", "loc_covid_id": "ORAL4844A2", "loc_admin_county": "Multnomah", "loc_admin_state": "OR", "is_disabled": false, "loc_name": "ONE MEDICAL DOWNTOWN", "created_date": "03-26-2021", "loc_admin_zip": "97204", "loc_admin_phone": "503-342-2520", "loc_admin_city": "Portland", "loc_admin_street1": "601 SW 4th Ave", "org_phone": "503-342-2520"}, "event_zip_code": 97204}
```

## Before Opening a PR
- [X] I tested this using the CLI (e.g., `poetry run vaccine-feed-ingest <state>/<site>`)
- [X] I ran auto-formatting: `poetry run tox -e lint-fix`

### CODE REVIEW

File: vaccine_feed_ingest/runners/or/getvaccinated_gov/parse.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+output_file = output_dir / "or_getvaccinated_gov.parsed.ndjson"
+
+
+with input_file.open() as fin:
+    data = json.load(fin)
+
+eastern = pytz.timezone("US/Eastern")
+# last_updated = eastern.localize(
+#     datetime.datetime.strptime(data["lastUpdated"], "%m/%d/%Y, %I:%M:%S %p")
+# ).isoformat()
```
Comment (eliblock) [inline vaccine_feed_ingest/runners/or/getvaccinated_gov/parse.py:23]: Is this needed?

**Review by eliblock — CHANGES_REQUESTED** [review]
Could this use the shared json->ndjson parser?

File: vaccine_feed_ingest/runners/or/getvaccinated_gov/parse.py (hunk 1 above)
Comment (MoralCode) [reply vaccine_feed_ingest/runners/or/getvaccinated_gov/parse.py:23]: probably a copy paste error tbh

**Comment by eliblock** [comment]
Closing all prs: https://blog.vaccinateca.com/vaccinate-the-states-is-winding-down

**Pull request closed without merging**